cloudant-client/src/main/java/com/cloudant/client/api/model/ExecutionStats.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2017 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.cloudant.client.api.model;
```

Comment:
I think this class should move into the `api.query` package (or maybe a `api.query.results` sub-package). Whilst it is part of the model I think we should follow the view pattern and keep the classes related to a particular kind of operation together as I think it should be easier for the end user (particularly in javadoc) to understand the classes they might need.

Reply:
See 96aec88